### DISCUSSION TITLE
How to use navigation-tree and navigation-sidebar simultaneously?

### DISCUSSION THREAD

**Discussion by auer-kk** [datenstrom/community#419] [Ask a question] [answered]
I would like to have both navigations simultaneously active on (wiki) pages. Any idea how to implement this the right or "yellow" way?

**Comment by markseuffert** [comment, marked as answer]
I don't know what's the best way. The first option is to implement your own navigation layout. Have a look at the existing navigations in the `system/layouts` folder. The second option is to use Markdown and put your sidebar in the `content/shared` folder. Some themes have support for header, footer and sidebar.

Hope this helps.
